Sources/StreamChat/Database/DTOs/ChannelDTO.swift

```diff
@@ -314,6 +330,24 @@ extension ChannelDTO {
     }
 }
 
+extension ChannelDTO {
+    func messagesArePopulated() -> Bool {
+        if lastMessageAt == nil {
+            return true
+        }
+
+        if oldestMessageAt == nil {
+            return true
+        }
+
+        let newestMessage = messages.max { a, b in
```

Comment:
Can't we assume the last is the newest message?

Reply:
messages is a `Set`